src/components/app.js

```diff
@@ -1,96 +1,47 @@
-import React from 'react';
-import { BrowserRouter as Router, Redirect, Switch, Route } from 'react-router-dom';
+import {
+  BrowserRouter as Router,
+  Switch,
```

Comment:
Looks like we've removed redirect from here. Could you explain why that is? Are we just then basically using routes instead of redirects (like, when someone tries to go to the onboard page, but should be directed to the dashboard)?

Reply:
There are redirects, technically, when these are all rendered. They are just being included elsewhere like in the `PrivateRoute` component.
The redirect you mentioned happens when a user tries to go to the onboard page but is redirected back to the dashboard is in the `onboard` component, I use the react hook `useHistory` to perform that kind of navigation. The only downside is you have to use functional components, which is no big deal I actually think they are lighter weight.